### ISSUE TITLE
[Bug]: Create from template inputs tag before template frontmatter

### ISSUE THREAD

**Issue by Unthetered** [RafaelGB/obsidian-db-folder#618]
### What happened?

1. I have a database folder from "tags"
2. When I create a new file from the table using "+" sign on the bottom left 
3. And use a template that already has frontmatter 
4. With templater templates and "trigger templater on new file creation"
5. It manually has the tag at the top of the file and corrupts the format
![Template](https://user-images.githubusercontent.com/118772607/203155789-cc424f51-301e-4d9e-b5f2-c2c6a00b2f49.JPG)
![create](https://user-images.githubusercontent.com/118772607/203155810-2236c48f-5c38-4ff0-ba1d-62549bfe0580.JPG)
![result](https://user-images.githubusercontent.com/118772607/203155815-de0c4a68-3b15-4f90-abab-4a13cdab517d.JPG)

I looked in the pluggin setting and in the database setting, cannot seem to make it work properly.

### What platform were you using?

Desktop

### Version of the plugin

2.8.4

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Commit 8293aa36

**Comment by Unthetered** [comment]
Yes or in the settings of the database there could be an option to add the tag could be added in the "tags" section of the front matter or at the bottom of the page instead of top (unless the issue is that the templater processing happens after DB folder adds the tag). This issue is only when generating a new file. When a file is already generated you can modify the tags and the frontmatter in the columns without issues.

**Comment by RafaelGB** [comment]
My opinion is that the inclusion of the tag is mandatory using that source to maintain consistency. If we give the option to remove it, it loses its meaning.

With the next version will be added at the end

About the Templater processing. Will be triggered once the file is created. The tag is added before
